### PR TITLE
Emit warning when Event not found

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/pipe-cd/pipe/issues/1497

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/eventwatcher/eventwatcher.go [hunk 1]
```diff
@@ -193,7 +197,7 @@ func (w *watcher) updateValues(ctx context.Context, repo git.Repo, events []conf
 		return nil
 	}
 
-	w.logger.Info(fmt.Sprintf("there are %d outdated values", len(commits)))
+	w.logger.Info(fmt.Sprintf("Event watcher will update %d outdated values", len(commits)))
```
Comment (nghialv) [inline pkg/app/piped/eventwatcher/eventwatcher.go:200]: nit: "event watcher"

Comment (nakabonne) [reply pkg/app/piped/eventwatcher/eventwatcher.go:200]: Right

**Comment by nghialv** [comment]
Nice.
/lgtm

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**